tests/pipelines/test_pipelines_image_segmentation.py

```diff
@@ -147,39 +147,60 @@ def test_small_model_tf(self):
         pass
 
     @require_torch
-    @unittest.skip("No weights found for hf-internal-testing/tiny-detr-mobilenetsv3-panoptic")
     def test_small_model_pt(self):
         model_id = "hf-internal-testing/tiny-detr-mobilenetsv3-panoptic"
 
         model = AutoModelForImageSegmentation.from_pretrained(model_id)
         feature_extractor = AutoFeatureExtractor.from_pretrained(model_id)
-        image_segmenter = ImageSegmentationPipeline(model=model, feature_extractor=feature_extractor)
```

Comment:
Just to check my understanding: we're testing two calls to the image_segmenter pipeline, one with one input image, the other with 2, and previously the "panoptic" task was tested and then "semantic" - whereas now both calls are for "semantic". Is the reason for this to make sure we're only testing one thing i.e. the pipeline can take batched and non-batched inputs?

Reply:
Batching is actually taken care somewhere else too for random models.

A list is not batched necessarily, but it should return a list of what could have been inferred individually.

`pipe([a, b]) == [pipe(a), pipe(b)]` in some sense.
Not all pipelines respect that unfortunately (the old ones mostly we never modified their behavior to keep things compatible).

The second test with the list was probably from somewhere where it was tested against and just updated I think.
The main property of `small_model_pt` is that we are checking real values that should be fixed for time eternal, meaning we should never deviate from the values, because that would be a breaking change.

It can happen that updates in the dependencies like `torch` or `Pillow` trigger changes in the output values, and that is fine (since we are not responsible for the change).
Any other kind of change is breaking backward compatiblity and should be done with caution and be documented as to why we're doing it. I think the current way of thinking is that the older something in the library is, the less likely it's ok to break anything with regards to it. If it was just added, then it's ok.

Reply:
>  previously the "panoptic" task was tested and then "semantic"

No previously only the `panoptic` was tested (it was defaulted to) for both.
But I'm switching here because `panoptic` appears to be broken, as it does not output anything (instead of outputting either 2 masks like before, or something resembling what `semantic` outputs).

Since I want to test against values that make sense I'm using `semantic` which seem OK.
When we find what's wrong with `panoptic` (or why the assumption that `panoptic` should return at least as much information as `semantic` is wrong) we can add them as tests again. 
And then this test would start checking both values, and that `panoptic` > `semantic` in expressiveness (which afaik  should always hold)

Reply:
Thank you for taking the time to write out such detailed explanations ❤️